models/userDao.js

```diff
@@ -0,0 +1,25 @@
+import prisma from '../prisma';
+
+const getUser = async (user_account) => {
+  return await prisma.$queryRaw(
+    `SELECT * FROM users WHERE user_account='${user_account}';`
+  );
+};
+
+const createUsers = async (
+  name,
+  email,
+  user_account,
+  phone_number,
+  hashedPw
+) => {
+  return await prisma.$queryRaw(`
+    INSERT INTO users(name, email, user_account, phone_number, password)
+    SELECT '${name}', '${email}', '${user_account}', '${phone_number}', '${hashedPw}'
+    WHERE NOT EXISTS 
+    (SELECT user_account FROM users 
+      WHERE user_account = '${user_account}')
+  `);
+};
+
```

Comment:
SQL 코드가 길어서 가독성있게 줄을 분리하시는 것이 좋겠습니다.
```suggestion

const createUsers = async (
  name,
  email,
  user_account,
  phone_number,
  hashedPw
) => {
  return await prisma.$queryRaw(`
    INSERT INTO users(
      name, 
      email,
      userAccount, 
      phoneNumber, 
      password)
    SELECT '${name}', '${email}', '${user_account}', '${phone_number}', '${hashedPw}'
    WHERE NOT EXISTS 
    (SELECT user_account FROM users 
      WHERE user_account = '${user_account}')
  `);
};

```
뒤에 `SELECT name, email user_account, phone_number, hashedPw`는 어떤 의도로 하시는건가요?

Reply:
소헌님 프리티어 이자식이 계속 저의 줄나누기를 방해하고있습니다 ... ㅜㅜㅜ

`SELECT name, email ...`은 SELECT로 데이터를 넣을 컬럼을 뽑아내서 insert를 하고싶었기 때문에 사용했습니다!!